<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="bulletComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>标准</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>项目符号(●)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>项目符号(○)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>项目符号(■)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>编号(1,2,3)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>编号(a,b,c)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>编号(A,B,C)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>编号(i,ii,iii)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>编号(I,II,III)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fontComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sizeComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actOpen"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actSave"/>
    <addaction name="actSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actPrint"/>
    <addaction name="actPrintPreview"/>
    <addaction name="separator"/>
    <addaction name="actexit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actUndo"/>
    <addaction name="actRedo"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actCut"/>
    <addaction name="actCopy"/>
    <addaction name="actPaste"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>格式</string>
    </property>
    <widget class="QMenu" name="menuZiti">
     <property name="title">
      <string>字体</string>
     </property>
     <addaction name="actBold"/>
     <addaction name="actItalic"/>
     <addaction name="actUnderline"/>
    </widget>
    <widget class="QMenu" name="menuAlign">
     <property name="title">
      <string>段落</string>
     </property>
     <addaction name="actLeft"/>
     <addaction name="actCenter"/>
     <addaction name="actRight"/>
     <addaction name="actJusitify"/>
    </widget>
    <addaction name="menuZiti"/>
    <addaction name="menuAlign"/>
    <addaction name="actColor"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="actClose"/>
    <addaction name="actCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actTile"/>
    <addaction name="actCascade"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actNext"/>
    <addaction name="actPrevious"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuWindow"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNew"/>
   <addaction name="actOpen"/>
   <addaction name="actSaveAs"/>
   <addaction name="actPrint"/>
   <addaction name="separator"/>
   <addaction name="actUndo"/>
   <addaction name="actRedo"/>
   <addaction name="actCut"/>
   <addaction name="actCopy"/>
   <addaction name="actPaste"/>
   <addaction name="separator"/>
   <addaction name="actBold"/>
   <addaction name="actItalic"/>
   <addaction name="actUnderline"/>
   <addaction name="separator"/>
   <addaction name="actLeft"/>
   <addaction name="actCenter"/>
   <addaction name="actRight"/>
   <addaction name="actJusitify"/>
  </widget>
  <action name="actNew">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="statusTip">
    <string>新建1个文档</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actPrint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="actPrintPreview">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打印预览</string>
   </property>
  </action>
  <action name="actexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actUndo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actRedo">
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="actCut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actBold">
   <property name="text">
    <string>加粗</string>
   </property>
  </action>
  <action name="actItalic">
   <property name="text">
    <string>倾斜</string>
   </property>
  </action>
  <action name="actUnderline">
   <property name="text">
    <string>下划线</string>
   </property>
  </action>
  <action name="actLeft">
   <property name="text">
    <string>左对齐</string>
   </property>
  </action>
  <action name="actCenter">
   <property name="text">
    <string>居中</string>
   </property>
  </action>
  <action name="actRight">
   <property name="text">
    <string>右对齐</string>
   </property>
  </action>
  <action name="actJusitify">
   <property name="text">
    <string>两端对齐</string>
   </property>
  </action>
  <action name="actColor">
   <property name="text">
    <string>颜色</string>
   </property>
  </action>
  <action name="actClose">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actCloseAll">
   <property name="text">
    <string>关闭所有</string>
   </property>
  </action>
  <action name="actTile">
   <property name="text">
    <string>平铺</string>
   </property>
  </action>
  <action name="actCascade">
   <property name="text">
    <string>层叠</string>
   </property>
  </action>
  <action name="actNext">
   <property name="text">
    <string>下一个</string>
   </property>
  </action>
  <action name="actPrevious">
   <property name="text">
    <string>前一个</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actexit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
